lib/modules/manager/dockerfile/readme.md

```diff
@@ -1,3 +1,8 @@
 Extracts all Docker images in a `Dockerfile`.
 
-If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+If Renovate does not update your Dockerfile images correctly, you may need to change the versioning format.
```

Comment:
If you search for Dockerfile in the docs production search, the first hit is this file. So we should probably explain that Renovate can't update all Dockerfiles out-of-the-box, because Docker lacks a proper versioning convention.

We also have more information about Docker versioning in the docs: https://docs.renovatebot.com/modules/versioning/#docker-versioning

The discussion mentions a `semver` setting like:

```json
{
  "packageRules": [
    {
      "matchPackageNames": ["whitesource/renovate"],
      "versioning": "semver"
    }
  ]
}
```

Do we want to add that example to the docs somewhere? Probably should go in the Docker versioning docs?

Reply:
I think it can go here, even if it doesn't apply to only Dockerfile